membership/src/test/java/net/corda/v5/membership/MemberInfoJavaApiTest.java

```diff
@@ -55,17 +55,17 @@ public void getName() {
     }
 
     @Test
-    public void getIdentityKeys() {
+    public void getLedgerKeys() {
```

Comment:
A little out of the scope of this, but while you are changing this file could you add a test to this class for getting the session initiation key also since it seems to be missing.